frontend/packages/dev-console/src/components/edit-application/EditApplicationForm.tsx

```diff
@@ -43,36 +44,43 @@ const EditApplicationForm: React.FC<FormikProps<FormikValues> & EditApplicationF
       <PageHeading title={createFlowType} />
```

Comment:
It should be enough to translate the title here? Wdyt?
```suggestion
      <PageHeading title={t(EditFlowPageTitle[createFlowType])} />
```